src/examples/pendulum.cpp

```diff
@@ -37,7 +37,7 @@ int main()
     // dmg.data = "arm_joint_3";
     // brk.push_back(dmg);
 
-    auto global_robot = std::make_shared<robot_dart::Robot>("res/models/pendulum.urdf");
+    auto global_robot = std::make_shared<robot_dart::robots::Robot>("pendulum.urdf");
```

Comment:
Why did you move `Robot.hpp` to a different namespace and folder?

Reply:
I wanted to make space for having the robot-specific classes (which in my mind was going with having the URDF organized). However, we should probably make a second PR for this.

Reply:
(by the way, git does not track the 'git mv' and therefore this creates conflict with any change to the Robot class in master... I did not expect this).

Reply:
Even if we have robot-specific classes, why would we need to move the base `Robot` class? I am okay with it, but it will break compatibility too much. I would keep the `Robot` class in the `robot_dart` namespace and move all the robot-specific classes in `robot_dart::robots` namespace.

Reply:
I wanted to follow your convention of having the abstract class in the directory/namespace of the concrete classes. You did this for control, for instance (robot_control.hpp). And we want to haves all the robots in a `robots` namespace.

Reply:
I would say that in this case `Robot` is not an abstract class, but a main ingredient of `robot_dart`. But in any case this is not a very big deal, I can move it before merging back to the main namespace.

Reply:
I can move it back if you prefer. It will make merging easier (btw, what do you use for indenting? I will need to reindent this file and it seems that my vs code does not do the same as yours)

Reply:
> I can move it back if you prefer.

Please do if you wish.

> btw, what do you use for indenting? I will need to reindent this file and it seems that my vs code does not do the same as yours

I am using clang-format and this format file: https://github.com/resibots/format_code/blob/master/.clang-format (just copy-paste it in your home folder).